### ISSUE TITLE
Suggestions for making Kiwix-desktop accessible for blind people

### ISSUE THREAD

**Issue by patrick-emmabuntus** [kiwix/kiwix-desktop#1083]
Hello Kiwix team,

I notice from Kiwix-desktop, is not accessible by Orca the free screen reader under Linux.

I made a video which shows the behavior of Orca with Kiwix, Kiwix-server and Firefox in narration mode: http://share.emmabuntus.org/Accessibility/Kiwix_vs_Kiwix_server_vs_Firefox_narrate.mp4

In this video, we analyze that Orca cannot read the content of the encyclopedia in Kiwix-desktop, while this is possible with Kiwix-server and Firefox.

So, it would be good for you to analyze why Orca-desktop content cannot be read by Orca. This may be the same with any other screen reader. If you want to test this, simply install Orca and launch it and open an application to see if it can read its content : https://ncu.libanswers.com/faq/366630

Regarding Firefox's narration mode, it is implemented by typing Alt+Ctrl+R then pressing N to start reading. Currently, it is the best mode for reading content for blind people because it does not read links, parentheses, and it tracks reading.

I hope that my comments will help you make Kiwix-desktop accessible to blind people.

**Comment by kelson42** [comment]
@patrick-emmabuntus Once an article is open should the user expect to get Orca reading the content autoatically without any action on the keyboard, or should the reading be made on demande via the keyboard?

**Comment by patrick-emmabuntus** [comment]
@kelson42 Yes Orca in principle automatically reads the contents of the window which has the focus. 

Otherwise do Orca + S to activate or deactivate the sound information. If you don't hear "Text-to-speech on or off". This is because the Orca configuration is not OK with speech synthesis. You must then open Orca and configure speech synthesis. Otherwise, to read an entire page, do Orca + down arrow.

PS: the Orca key is either the insert key on a desktop or the caps lock key on a laptop.

In a terminal to : 
- start Orca : orca -r
- configure Orca : orca -s -r

**Comment by kelson42** [comment]
@patrick-emmabuntus I can reproduce the problem and this is indeed imparing... and suprisingly failing. I will have a look.

**Comment by kelson42** [comment]
The problem has not been automatically fixed by moving to Qt6. We will have to do something. Actually the problem is not only the browser, IMHO it seems that Orca if far too verbose about technical stuff . Here a few links which might help:
* https://wiki.gnome.org/Accessibility/ATK/BestPractices
* https://doc.qt.io/qt-6/accessible-qwidget.html
* https://doc.qt.io/qt-6/accessible-qtquick.html
